packages/create/src/recipes/nextjs/index.ts

```diff
@@ -102,7 +102,7 @@ const recipe: Recipe<NextjsOptions> = {
         lint: "next lint",
       },
       dependencies: {
-        ...(useEdgeDBAuth ? { "@edgedb/auth-nextjs": "^0.1.0" } : {}),
+        ...(useEdgeDBAuth ? { "nuxt-edgedb-module": "^0.1.0" } : {}),
```

Comment:
I think this is an accidental change, right?

Reply:
Oh yeah it is; 🤦‍♂️